terraform/s3.tf

```diff
@@ -0,0 +1,36 @@
+resource "aws_s3_bucket" "backoffice-static" {
+  bucket        = "${module.beacons_label.name}-${module.beacons_label.environment}"
+  tags          = module.beacons_label.tags
+  force_destroy = true
+  acl           = "public-read"
+
+  cors_rule {
+    allowed_headers = ["*"]
+    allowed_methods = ["PUT", "POST"]
+    allowed_origins = ["*"]
+    expose_headers  = ["ETag"]
+    max_age_seconds = 3000
+  }
```

Comment:
Need some help with this.  What's the proper CORS configuration for an S3 static site calling cross-origin APIs?